Dwifft/Dwifft+UIKit.swift

```diff
@@ -83,7 +90,10 @@ open class CollectionViewDiffCalculator<T: Equatable> {
 
                     self.collectionView?.insertItems(at: insertionIndexPaths)
                     self.collectionView?.deleteItems(at: deletionIndexPaths)
-                }, completion: nil)
+                
+                }, completion: { (completion) in
```

Comment:
@anneWe On this line, the boolean parameter `completion` isn't being used. Can we change the signature to `_ in` to make it more obvious that it's not used?

[More information](https://developer.apple.com/reference/uikit/uicollectionview/1618045-performbatchupdates) on the purpose of this parameter.